backend/prisma/schema.prisma

```diff
@@ -0,0 +1,80 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+enum report_status {
+  SEEN
+  UNSEEN
+  REMOVED
+  SETTLED
+}
+
+model courses {
+  courseCode         String     @id(map: "pk_course_code") @map("course_code")
+  archived           Boolean
+  attributes         String[]
+  calendar           String
+  campus             String
+  description        String
+  enrolmentRules     String     @map("enrolment_rules")
+  equivalents        String[]
+  exclusions         String[]
+  faculty            String
+  fieldOfEducation   String     @map("field_of_education")
+  genEd              Boolean     @map("gen_ed")
+  level              Int
+  school             String
+  studyLevel         String     @map("study_level")
+  terms              Int[]
+  title              String
+  uoc                Int
+  rating             Float
```

Comment:
lets remove this field

Reply:
would that be fine for the database? I wasn't sure if it would break all the old data or reviews so i didnt touch the actual schema

Reply:
only database we have right now is the staging one, which is fine for us to delete and redo again

Reply:
yeah so just removing the rating field means migrations from migration server doesnt work.

Also now that i have a closer look, migration server is still hard coded up with typeORM 😭, i think we'll need to remove the orm from the migration server so we can start flexibly changing the db structure without pain.